src/core/seed.js

```diff
@@ -44,6 +48,19 @@ Seed.prototype.parse_json = function(j) {
   return this;
 };
 
+Seed.prototype.parse_base = function(j) {
+  if (typeof j !== 'string') {
+    throw new Error('Value must be a string');
+  }
+  if (!j.length || j[0] !== 's') {
+    this._value = NaN;
+  } else {
+    this._value = Base.decode_check(Base.VER_FAMILY_SEED, j);
```

Comment:
Instead of making a new function, could we just directly use `Base.decode_check(Base.VER_FAMILY_SEED, j);`